sessionctx/variable/session.go

```diff
@@ -712,6 +735,23 @@ func (s *SessionVars) IsAutocommit() bool {
 	return s.GetStatusFlag(mysql.ServerStatusAutocommit)
 }
 
+// IsReadConsistencyTxn if true it means the transaction is an read consistency (read committed) transaction.
+func (s *SessionVars) IsReadConsistencyTxn() bool {
+	var isoLevel string
+	if s.TxnIsolationLevelOneShot.State == 2 {
```

Comment:
The `2` seems a bit little difficult  to understand, maybe we should refactor this later

Reply:
Yes, I just revert the changes in https://github.com/pingcap/tidb/pull/7282 .

Reply:
done